app/models/export.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 class Export < ApplicationRecord
   self.table_name = :exports
-  belongs_to :resource, polymorphic: true
-  belongs_to :external_system
+  belongs_to :resource, polymorphic: true, optional: true
+  belongs_to :external_system, optional: true
```

Comment:
I don't understand this change ?  The export should be deleted - i can't see a use case where an export can exist without a resource or external system - it would be a useless record

Reply:
My mistake - I thought `optional: true` prevents associated resources from being deleted. Removed now